.github/workflows/build.yml

```
@@ -13,11 +13,11 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        cxx: [11,14,17, 20]
+        cxx: [17,20]
```

```suggestion
        cxx: [17, 20]
```
Yeah I think it's totally fine to drop C++11 and C++14. People being interested in boxed-cpp most likely won't depend on a too old compiler anyways.